scripts/ci/community_csi_test_cleanup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -xe
 
-if [ `docker ps | grep $1 | wc -l` != 0 ] ; then
-    docker stop $1
+if [ `docker ps -a | grep $1 | wc -l` != 0 ] ; then
+    docker rm -f $1
 fi
```

Comment:
is this because of some new gap?
could you add a newline at EOF while you're at it?

Reply:
and did the problem occur with `-a` but without `-f`? because it would be better to avoid `-f` if possible

Reply:
didn't check it, @matancarmeli7 what do you think ?

Reply:
@matancarmeli7 answer through slack: 
"we are using -a in order to get also containers with the wanted name that are also stopped, because they are interrupt when you are trying to create a container with the same name as they do.
About the -f it's important because you want to delete running containers with the wanted name that can fail your run"

Reply:
I now consulted with @matancarmeli7 as well, we agree on `docker stop` + `docker rm` instead of `-f`, to avoid an error-prone SIGKILL, while not relying on existing `--rm`